docs/sw-cli.md

```diff
@@ -36,6 +36,21 @@ After the command has run you'll have two new files, a
 `sw-lib.vX.X.X.min.js` and `sw.js` file (unless you changed
 the file name).
 
+## Creating a Config File
+
+One of the questions asked by `sw-cli generate:sw` is whether you'd like to save
+your responses to a configuration file. Answering `Y` will save your responses
+to `sw-cli-config.json`. Subsequent runs of the cli will regenerate the service
+worker based on the config file, which makes building faster. 
+
+If you don't need anything more complicated (in other words if you don't need
```

Comment:
I don't see using a `gulp` plugin that shells out to `sw-cli` as being a build process that we'd recommend. This paragraph could just be cut.

There's also no reason to assume that someone reading the docs for `sw-cli` is using `gulp`, so if you did feel like this paragraph needs to stay, I wouldn't make it `gulp`-specific.